x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks.tsx

```diff
@@ -126,10 +128,14 @@ const BenchmarkSearchField = ({
 };
 
 export const Benchmarks = () => {
+  const [pageSizes, setPageSize] = useLocalStorage<number>(
+    LOCAL_STORAGE_PAGINATION_BENCHMARK_KEY,
+    10
+  );
   const [query, setQuery] = useState<UseCspBenchmarkIntegrationsProps>({
     name: '',
     page: 1,
-    perPage: 10,
+    perPage: pageSizes!,
```

Comment:
it looks like you're already passing a `number` type to `useLocalStorage`'s generic, do we still need the non-null assertion after that? 
maybe a better solution would be to set a default value of 10 in case `pageSize` is undefined